test/decompile/basic.txt

```diff
@@ -105,6 +105,11 @@
     i32.const 0  ;; fi
     call_indirect
     i32.const 0
+    if
+    else
+      nop
```

Comment:
Should we verify the empty else too?

Reply:
It seems that empty else statements just completely get removed, shall I add it anyway for if that ever changes?

Reply:
Yes, I think maybe removing the nop might give a little more coverage, which includes that case, right?